### ISSUE TITLE
Ideas to modify journal

### ISSUE THREAD

**Issue by catdad666** [whymog/efjm#46]
I know we maybe planned to do more with the journal in the second half, but as the game stands now it gets an "obtained" moment, but never serves a purpose. Is there anything we can do with the journal to make it more useful in this half of the game? Maybe the player opens it when triggering the flashbacks?

My other thought was changing it to a sketchbook and having it serve as an image gallery, but then it would have to be a constant option and that seems hard to rig in Twee/Twine.

Open to any and all thoughts!

**Comment by whymog** [comment]
I think those are some good ideas to consider for a "complete" game, but I'm struggling to think of a good way to hook it into the existing game as it stands. However, as more feedback comes in, I think we should keep this in our back pocket as a possible vehicle to clear up any confusion or bridge gaps that players may experience.

**Comment by catdad666** [comment]
I worked up a quick fix for this in my Mom's Tent edits last night. After you find the journal, you get the map by examining a paper sticking out of it. Not ideal, IMO. But, it's a little more of a use that doesn't require as much writing (journal = flashback trigger) or wild code solutions (journal = sketchbook).

We have another one-time use obtained item, the lighter, which also just reveals a new image (and your surroundings) so using the journal as a map-revealing tool fits with that. Closing this for now.
